components/Mermaid/index.jsx

```diff
@@ -0,0 +1,32 @@
+const { default: mermaid } = require('mermaid/dist/mermaid.esm.mjs');
```

Comment:
How big is this? I wonder if there's a way we can lazily load this in, so it doesn't blow up the bundle size for everyone, regardless of whether they're using mermaid or not?

Reply:
It's **very** big (~120kb). It def triggered the build flags.